package/main/daml/Daml.Finance.Account/daml.yaml

```diff
@@ -13,7 +13,7 @@ data-dependencies:
   - .lib/daml-finance/Daml.Finance.Interface.Holding/1.0.0/daml-finance-interface-holding-1.0.0.dar
   - .lib/daml-finance/Daml.Finance.Interface.Types.Common/1.0.0/daml-finance-interface-types-common-1.0.0.dar
   - .lib/daml-finance/Daml.Finance.Interface.Util/1.0.0/daml-finance-interface-util-1.0.0.dar
-  - .lib/daml-finance/Daml.Finance.Util/2.0.0/daml-finance-util-2.0.0.dar
+  - .lib/daml-finance/Daml.Finance.Util/1.0.0.99.202323.1/daml-finance-util-1.0.0.99.202323.1.dar
```

Comment:
I guess that `Daml.Finance.Account` (and all packages whose dependencies have changed) should also be bumped to a pre-release version.

Reply:
And bump the minor to `99`, i.e., `1.99.x`